.vscode/settings.json

```diff
@@ -16,6 +16,7 @@
         "titleBar.activeBackground": "#648FFF",
         "titleBar.activeForeground": "#000000"
 	},
+	"jest.jestCommandLine": "fnm use; yarn workspace @guardian/dotcom-rendering test",
```

Comment:
This is actually part of #4850

Reply:
I'm a bit concerned about seen `fnm` here and all our docs advise `nvm` and I would assume this won't run on my machine since I don't have `fnm` instaled?

Reply:
Good shout. Have removed from this PR, and updated #4850 to skip the node manager step.

Reply:
One for a client side infra meeting?!

Reply:
We could always write a wrapper in the makefile which allows us to run 'use' on nvm or fnm (or even asdf if we wanted)